cmd/config/subcommand/sandbox/sandbox_config.go

```diff
@@ -3,7 +3,7 @@ package sandbox
 //go:generate pflags Config --default-var DefaultConfig
 var (
 	DefaultConfig = &Config{
-		Debug: true,
+		Debug: false,
```

Comment:
Why is this changed back to false.

Reply:
I removed debug flag. I didn't find any useability if we are using true as the default value. It will become more confusing